cli/file_fetcher.rs

```diff
@@ -32,9 +32,6 @@ use std::sync::Arc;
 use std::sync::Mutex;
 
 /// Structure representing local or remote file.
-///
-/// In case of remote file `url` might be different than originally requested URL, if so
-/// `redirect_source_url` will contain original URL and `url` will be equal to final location.
```

Comment:
Why remove this comment?

Reply:
Because there is no `redirect_source_url`.  It was refactored out at some point and no one bothered to update the comment.